holoviews/plotting/bokeh/element.py

```diff
@@ -538,17 +538,44 @@ def __init__(self, element, plot=None, **params):
 
     def initialize_plot(self, ranges=None, plot=None, plots=None):
         self.mplplot.initialize_plot(ranges)
-        plot = mpl.to_bokeh(self.mplplot.state)
+
```

Comment:
How many ``BokehMPLWrapper`` plots are there? These plots depend on both matplotlib **and** bokeh. This isn't typically a problem but I would like to know how many of these there are.